lib/ddtrace/configuration/settings.rb

```diff
@@ -545,16 +545,35 @@ def initialize(*_)
 
         # TODO: This setting is not tracer-specific and should be moved to top-level or to the transport.
         option :port
-        option :priority_sampling # TODO: Deprecate
+
+        # Enables {https://docs.datadoghq.com/tracing/trace_retention_and_ingestion/#datadog-intelligent-retention-filter
+        # Datadog intelligent retention filter}.
+        # @default `true`
+        # @return [Boolean,nil]
+        option :priority_sampling
```

Comment:
I'm curious -- why did we end up not deprecating this?

Reply:
Because enabling and disabling intelligent retention filter is actually someone we want to expose :)

The deprecation intention was a mistake.